pageserver/src/http/routes.rs

```diff
@@ -1689,8 +1689,20 @@ where
                 let token_cloned = token.clone();
                 let result = handler(r, token).await;
                 if token_cloned.is_cancelled() {
-                    info!("Cancelled request finished");
+                    // dropguard has executed: we will never turn this result into response.
+                    //
+                    // at least temporarily do debug logging; these failures are rare enough but
```

Comment:
It's doing `info!` and `error!` logging though.

Reply:
I meant the `{:?}` with the error, should clarify.

Reply:
```suggestion
                    // at least temporarily do {:?} logging; these failures are rare enough but
```